deployment/refinery-modules/refinery/manifests/docker_.pp

```diff
@@ -0,0 +1,5 @@
+class refinery::docker_ {
+  class { 'docker':
```

Comment:
Yes, it's the reason you got the naming error when trying to use `refinery::docker` on the line above. Here's the source in Puppet docs (still relevant as of Puppet v3.5 and we are using v3.4.3): https://docs.puppet.com/puppet/3.5/lang_namespaces.html#workaround